cli/src/main/java/com/box/l10n/mojito/cli/filefinder/file/YAMLFileType.java

```diff
@@ -0,0 +1,10 @@
+package com.box.l10n.mojito.cli.filefinder.file;
+
+/** @author loliveiramontedonio */
+public class YAMLFileType
```

Comment:
to be consistent with the filter name, can you adjust the case "YAML" --> "Yaml"